tests/unit/src/kakarot/accounts/eoa/test_library.py

```diff
@@ -28,49 +28,63 @@ async def externally_owned_account(starknet, mock_kakarot):
 @pytest.mark.asyncio
 class TestLibrary:
     async def test_execute_should_make_all_calls_and_return_concat_results(
-        self, externally_owned_account
+        self, mock_externally_owned_account, eth
     ):
         private_key = generate_random_private_key()
-        calls = []
-        calldata = b""
-        expected_result = []
-        for transaction in TRANSACTIONS:
-            tx = Account.sign_transaction(transaction, private_key)["rawTransaction"]
-            calls += [
-                (
-                    0x0,  # to
-                    0x0,  # selector
-                    len(calldata),  # data_offset
-                    len(tx),  # data_len
-                )
-            ]
-            calldata += tx
-            # Execute contract bytecode
-            if transaction["to"] != "":
-                expected_result += [
-                    int(transaction["to"], 16),
-                    transaction["value"],
-                    transaction["gas"],
-                    len(
-                        bytes.fromhex(transaction.get("data", "").replace("0x", ""))
-                    ),  # calldata_len
-                    *list(
-                        bytes.fromhex(transaction.get("data", "").replace("0x", ""))
-                    ),  # calldata
-                ]
-            # Deploy Contract
-            else:
-                expected_result += [
-                    len(
-                        bytes.fromhex(transaction.get("data", "").replace("0x", ""))
-                    ),  # calldata_len
-                    *list(
-                        bytes.fromhex(transaction.get("data", "").replace("0x", ""))
-                    ),  # calldata
-                ]
+        (calls, calldata, expected_result) = get_multicall_from_evm_txs(
+            TRANSACTIONS, private_key
+        )
+        total_transferred_value = sum([x["value"] for x in TRANSACTIONS])
```

Comment:
unused here

Reply:
I need `total_transferred_value` in order to mint this amount to the `mock_externally_owned_account`, otherwise the execution of the multicall will not work (since they include a value).